pkg/ccl/backupccl/restore_job.go

```diff
@@ -706,6 +687,26 @@ func restore(
 	g.GoCtx(func(ctx context.Context) error {
 		ctx, progressSpan := tracing.ChildSpan(ctx, "progress-log")
 		defer progressSpan.Finish()
+		if !dataToRestore.isMainBundle() {
```

Comment:
any reason not to put the whole `GoCtx` inside the conditional ?

Reply:
Nope. Moved the if outside after the rebase.